images/linux/toolsets/toolset-1804.json

```diff
@@ -68,6 +68,11 @@
             ]
         }
     ],
+    "tools":[
```

Comment:
I don't think we need this section. We can take the tools from the `toolcache` block that have `"url"` property

Reply:
Agree with Mikhail. We don't need the separate section.
We can just take the tools with `url` property. Or add separate flag property as an alternative.